### DISCUSSION TITLE
Can we add metadata into the existing "metadata" jsonb column in storage.objects table?

### DISCUSSION THREAD

**Discussion by kk21** [supabase/supabase#4112] [Questions] [answered]
Hi,

Can we add metadata into the existing "metadata" jsonb column in storage.objects table?

Or should we save our metadata in our own table?

Thanks.

**Comment by fenos** [comment, marked as answer]
Currently, it's not possible to add custom metadata to the object metadata column.
It is advisable to store that metadata in a sperate table.

If you have a good use case to allow for custom metadata on the storage column, feel free to raise an issue https://github.com/supabase/storage-api/issues
